### PR TITLE
Extract new strings (2024-06-27)

### PR DESCRIPTION
**Pull request by mozilla-pontoon**
Automated changes by [create-pull-request](https://github.com/peter-evans/create-pull-request) GitHub action

### CODE REVIEW

**Review by Delphine — APPROVED** [review]

**Review by Delphine — CHANGES_REQUESTED** [review]
Woops, missed the error called out by the linter

File: af/firefox-ios.xliff [hunk 1]
```diff
@@ -3995,6 +4011,10 @@ Private Tab</source>
         <source>Very dissatisfied</source>
         <note>On the microsurvey, this is the title for one of the options that the user can select to answer the survey.</note>
       </trans-unit>
+      <trans-unit id="Microsurvey.Survey.Options.LikertScaleOption6.v129" xml:space="preserve">
+        <source>I don't use it</source>
```
Comment (Delphine) [inline af/firefox-ios.xliff:4015]: @cyndichin Can you please update the apostrophe so it's a typographic apostrophe?
We should use that in the strings (not necessary in l10n comments)

**Review by Delphine — CHANGES_REQUESTED** [review]

File: af/firefox-ios.xliff (hunk 1 above)
Comment (cyndichin) [reply af/firefox-ios.xliff:4015]: thanks! will update! Is there documentation on this so I can link it to my PR? I will update it to `I don’t use it` let me know if there's anything else I need.

Comment (cyndichin) [reply af/firefox-ios.xliff:4015]: PR has been merged.

**Review by flodolo — APPROVED** [review]
Thanks. I'll merge, since this was the only issue highlighted by Delphine, and it's now fixed.

File: af/firefox-ios.xliff (hunk 1 above)
Comment (Delphine) [reply af/firefox-ios.xliff]: @cyndichin Yes there is documentation [here](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#land-good-quality-strings) about this.

Generally speaking, this entire documentation is comes in very handy when working with strings: [Localization Best Practices for Developers](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html).

It comes in really handy for engineers who start working with localization files, so feel free to share the love in those cases! 😃 thank you!

**Pull request merged**